into/utils.py

```diff
@@ -11,6 +12,11 @@
 from .compatibility import unicode
 
 
+def ext(filename):
+    _, e = os.path.splitext(filename)
+    return e.lstrip(os.extsep)
```

Comment:
We use `ext(filename) == 'gz'` in a few places, so changing that behavior would break some things.